Charts/base-Django/templates/app/secret.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "django.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "django.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{ toYaml .Values.commonLabels | nindent 4 }}
+    {{- end }}
+    app.kubernetes.io/component: django
+  annotations: 
+{{- if .Values.commonAnnotations }}
+{{- toYaml .Values.commonAnnotations | nindent 4 }}
+{{- end }}
+    "helm.sh/hook": pre-upgrade, pre-install
+    "helm.sh/hook-weight": "0"
+data:
+  DJANGO_JWT_PRIVATE_SIGNING_KEY: {{ ( default "" .Values.DjangoJwtPrivateSigningKey ) | quote | b64enc }}
```

Comment:
this JWT secret is specific to some projects only

Reply:
for this I created a variable djangoAppSecrets, that will allow users to add all the additional required secrets for their app, so in their applications chart, in their values file, it will look something like this.
``` 
django:
  djangoAppSecrets:
    - key: DJANGO_SECRET_KEY
      value: ThisIsAnExampleKeyForTest
    - key: DJANGO_JWT_PRIVATE_SIGNING_KEY
      value: <JWT_SIGNING_KEY>
```

Reply:
nice!